cudamapper/src/index_gpu.cuh

```diff
@@ -29,6 +28,10 @@
 #include <claragenomics/utils/mathutils.hpp>
 #include <claragenomics/utils/signed_integer_utils.hpp>
 
+////////////////
+#include <iostream>
+////////////////
```

Comment:
Is `iostream` still needed?

Reply:
no, `////////////////` was supposed to catch my attention :man_facepalming:

Reply:
done